libnavui-maps/src/test/java/com/mapbox/navigation/ui/maps/guidance/junction/JunctionProcessorTest.kt

```diff
@@ -299,17 +299,18 @@ class JunctionProcessorTest {
 
     @Test
     fun `process action junction process raster to bitmap failure`() {
-        mockkStatic(MapboxRasterToBitmapParser::class)
-        val mockData = byteArrayOf()
-        val action = JunctionAction.ParseRasterToBitmap(mockData)
-        every { MapboxRasterToBitmapParser.parse(any()) } returns
-            ExpectedFactory.createError(
-                "Error parsing raster to bitmap as raster is empty"
-            )
-
-        val result = JunctionProcessor.process(action) as JunctionResult.JunctionBitmap.Failure
-
-        assertEquals("Error parsing raster to bitmap as raster is empty", result.message)
+        mockkStatic(MapboxRasterToBitmapParser::class) {
+            val mockData = byteArrayOf()
+            val action = JunctionAction.ParseRasterToBitmap(mockData)
+            every { MapboxRasterToBitmapParser.parse(any()) } returns
+                ExpectedFactory.createError(
+                    "Error parsing raster to bitmap as raster is empty"
+                )
+
+            val result = JunctionProcessor.process(action) as JunctionResult.JunctionBitmap.Failure
+
+            assertEquals("Error parsing raster to bitmap as raster is empty", result.message)
+        }
```

Comment:
Can we add another test that would verify this new error message `ExpectedFactory.createError("Raster is not a valid bitmap")`
@Zayankovsky

Reply:
There is already such a test in `MapboxRasterToBitmapParserTest`.